cirq-google/cirq_google/common_serializers.py

```diff
@@ -652,6 +652,66 @@ def _can_serialize_limited_iswap(exponent: float):
 #
 #############################################
 
+#
+# Coupler Pulse serializer and deserializer
+#
+
+COUPLER_PULSE_SERIALIZER = op_serializer.GateOpSerializer(
+    gate_type=CouplerPulse,
+    serialized_gate_id='pulse',
+    args=[
+        op_serializer.SerializingArg(
+            serialized_name='coupling_mhz', serialized_type=float, op_getter='coupling_mhz'
```

Comment:
Internally we tend to use the standard case for units, even when it clashes with case conventions, so this would be `coupling_MHz`.

Reply:
This violates case conventions per Google python style guide, so I don't think it is a good convention to follow.

Reply:
+1 for lowercase